back/engines/commercial/multi_tenancy/config/intercom-event-whitelist.txt

```diff
@@ -23,8 +23,6 @@ Idea changed_body
 Idea changed_status
 Idea changed_title
 Idea deleted
-Idea first published by user
-Idea first_published_by_user
```

Comment:
I'm not sure what we use intercom for here, so I'm kind of guessing we don't need these entries anymore.